docs/trusted-resources.md

```diff
@@ -148,6 +148,46 @@ To learn more about `ConfigSource` please refer to resolvers doc for more contex
  * enforce (default) - fail the taskrun/pipelinerun if verification fails
  * warn - don't fail the taskrun/pipelinerun if verification fails but log a warning
 
+#### TaskRun and PipelineRun status update
+
+Trusted resources will update the taskrun/pipelinerun’s [condition](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties) to indicate if it passes verification or not.
+
+**No Matching Policies:**
+
+|                             | `Conditions.TrustedResourcesVerified` | `Conditions.Succeeded` |
+|-----------------------------|---------------------------------------|------------------------|
+| `no-match-policy`: "ignore" |                                       |                        |
```

Comment:
Does this mean the condition is not present? or that it's present, but doesn't have a value set?

Reply:
not exactly, this mean the code doesn't update this condition. For this case, the `TrustedResourcesVerified` is not present but `Succeeded` condition may be of any status

Reply:
I will update this table to clarify.

Reply:
I added one note above the table, wonder if this is ok?
```
Note that if not `true` or `false` means this case doesn't update the corresponding condition.
```

Reply:
ah ok that's fine!